db/main/migrate/20180222000002_add_migrating.rb

```diff
@@ -0,0 +1,9 @@
+class AddMigrating < ActiveRecord::Migration
+  def change
+    add_column :repositories, :migrating, :boolean
+    add_column :repositories, :migrated_at, :datetime
+
+    add_column :organizations, :migrating, :boolean
+    add_column :users, :migrating, :boolean
```

Comment:
SO we don't need `migrated_at` for `users` at all? Are we only going to migrate data per repo?

Reply:
OK, I see it is added in a later migration in this PR.